junit-tests/src/test/java/org/junit/gen5/api/AssertionsTests.java

```diff
@@ -531,8 +538,59 @@ void assertEqualsFloatWithUnequalValuesAndMessageSupplier() {
 	}
 
 	@Test
+	void assertEqualsFloatWithDelta() {
+		assertEquals(0.56f, 0.6f, 0.05f);
+		assertEquals(0.01f, 0.011f, 0.002f);
+		assertEquals(Float.NaN, Float.NaN, 0.5f);
+	}
+
+	@Test
+	void assertEqualsFloatWithDeltaWithUnequalValues() {
+		AssertionFailedError e1 = expectThrows(AssertionFailedError.class, () -> assertEquals(0.5f, 0.2f, 0.2f));
+		assertMessageEndsWith(e1, "expected: <0.5> but was: <0.2>");
+
+		AssertionFailedError e2 = expectThrows(AssertionFailedError.class, () -> assertEquals(0.1f, 0.2f, 0.000001f));
+		assertMessageEndsWith(e2, "expected: <0.1> but was: <0.2>");
+
+		AssertionFailedError e3 = expectThrows(AssertionFailedError.class, () -> assertEquals(100.0f, 50.0f, 10.0f));
+		assertMessageEndsWith(e3, "expected: <100.0> but was: <50.0>");
+
+		AssertionFailedError e4 = expectThrows(AssertionFailedError.class, () -> assertEquals(-3.5f, -3.3f, -42.0f));
+		assertMessageEndsWith(e4, "expected: <-3.5> but was: <-3.3>");
```

Comment:
This is probably a good idea. JUnit 4 javadoc even says that delta should be positive but code does not check this. I'll add such check.